create_rental.rb

```diff
@@ -0,0 +1,35 @@
+require './input'
+require './rental'
+
+# CreateRental
+class CreateRental
```

Comment:
Hello @GzimAsani  I think you were inputting index number instead of person id, because if you insert person id you get the rentals back: 

![image](https://user-images.githubusercontent.com/77038610/152826298-cb6bee12-4d0c-409b-b0e6-9b483d5bd54b.png)